rtil/src/native/character.rs

```diff
@@ -127,6 +128,25 @@ impl AMyCharacter {
             fun.call(obj.as_ptr(), &params);
         }
     }
+    pub fn camera_mode() -> u8 {
+        unsafe {
```

Comment:
unnecessary unsafe block (it even says so in the warnings -- don't ignore all warnings :) )